src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java

```diff
@@ -204,7 +204,7 @@ public String createAnalysisSummary(FormatterFactory formatterFactory) {
                                                          Optional.ofNullable(newCoverage).map(decimalFormat::format)
                                                                  .map(i -> i + "% Coverage")
                                                                  .orElse("No coverage information") + " (" +
-                                                         decimalFormat.format(coverage) + "% Estimated after merge)")),
```

Comment:
Nitpick: Avoid using negation in ternaries, as it makes the logic difficult to follow. I'd probably switch this to `Optional.ofNullable(coverage).orElse(0)` which does incur the overhead of an extra object on the heap, but helps makes the code more readable.